spec/audited/sweeper_spec.rb

```diff
@@ -1,6 +1,12 @@
 require "spec_helper"
 
 class AuditsController < ActionController::Base
+  if Rails::VERSION::MAJOR == 4
+    before_filter :populate_user
```

Comment:
this will cause deprecations on 4.2 ... and I raise on deprecations ... so kaboom :D 
... also add the raise on deprecations option to find this in tests easier.